bsp/fh8620/drivers/pwm.c

```diff
@@ -186,12 +186,14 @@ int fh_pwm_probe(void *priv_data)
     PWM_Enable(pwm_obj, RT_FALSE);
 
     pwm_dev = rt_malloc(sizeof(struct rt_device));
```

Comment:
why not rt_calloc?

Reply:
Thanks for your advise, it is a more better  idea to fix this bug, I had updated this bug with your advise.